general/solomiia_tymoshchuk/lecture3/_3/Triangle.java

```diff
@@ -0,0 +1,27 @@
+package general.solomiia_tymoshchuk.lecture3._3;
+
+public class Triangle implements Shape {
+
+    double side;
+    public Triangle(double side){
+        this.side = side;
+
+    }
+
+    @Override
+    public double findPerimeter() {
+        double perimeter = side*3;
```

Comment:
value can be returned directly from the method without creation of a local variable (since it's clear from the method name what it represents)